src/xpk/core/blueprint/blueprint_generator.py

```diff
@@ -120,6 +127,10 @@ def generate_a3_mega_blueprint(
                 "display_name": "kubectl-access-network",
             }],
             "system_node_pool_machine_type": system_node_pool_machine_type,
+            "system_node_pool_node_count": {
+                "total_min_nodes": system_node_pool_min_node_count,
+                "total_max_nodes": 1000,
```

Comment:
1000 should be saced to global var

Reply:
I wouldn't make variables for template constant values.